<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cilent</class>
 <widget class="QWidget" name="Cilent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <widget class="QLineEdit" name="LeName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LXQ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbConnect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QListWidget" name="LwShowFriend">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>101</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="LwLog">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>141</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LReceive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>接收：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TeSendMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbSend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>41</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QListWidget" name="LwShowMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>331</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>34</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>好友列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>34</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>日志记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户姓名：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
